.gitmessage

```diff
@@ -29,3 +29,4 @@
 # Co-authored-by: Marco Strigl <mstrigl@suse.com>
 # Co-authored-by: Michael Schroeder <mls@suse.de>
 # Co-authored-by: Frank Schreiner <schreiner@suse.de>
+# Co-authored-by: Dario Leidi <dleidi@suse.de>
```

Comment:
This doesn't match the address Dario is using in commits....

Reply:
Thanks for checking it out. It's updated